solana-programs/staking-bridge/tests/staking-bridge.ts

```diff
@@ -1,16 +1,293 @@
-import * as anchor from "@coral-xyz/anchor";
-import { Program } from "@coral-xyz/anchor";
-import { StakingBridge } from "../target/types/staking_bridge";
+import * as anchor from '@coral-xyz/anchor'
```

Comment:
filename here is snake-case, but utils are camelCase

Reply:
this filename is created by anchor when you init. i can change the others to be snake case

Reply:
actually i'll leave as-is if no big deal to you as our js/ts files are usually camelCased and I also dont want to change anchor's default casing for the test file